js/views/playingScreen.js

```diff
@@ -177,6 +203,27 @@ var PlayingScreen = React.createClass({
     this.props.controller.handleMuteClick();
   },
 
+  /**
+   *
+   * @param vrKey - key for durection in config
```

Comment:
durection -> Is this direction or duration?

Reply:
Also, please rename the variable to to vrDirection or vrDuration.  Although it's used as a key, whoever is using this function shouldn't know that.

Reply:
\+